### DISCUSSION TITLE
Vaillant Heatpump searching for more values

### DISCUSSION THREAD

**Discussion by JanHBade** [john30/ebusd#629] [Q&A]
Hello,

i want reading the "cold" and the "hot" temperature  of the cooling fuel and the flow rate, any suggestion here?

ebusctl info
```
address 03: master #11
address 08: slave #11, scanned "MF=Vaillant;ID=HMU00;SW=0305;HW=0403", loaded "vaillant/08.hmu.csv"
address 10: master #2
address 15: slave #2, scanned "MF=Vaillant;ID=70000;SW=0419;HW=4603", loaded "vaillant/15.700.csv"
address 31: master #8, ebusd
address 36: slave #8, ebusd
address 71: master #9
address 76: slave #9, scanned "MF=Vaillant;ID=VWZ00;SW=0305;HW=0403"
```

**Comment by mwildbolz** [comment]
Maybe you find some additional parameters in my config.
See for example [my repo](https://github.com/mwildbolz/ebusd-configuration/blob/f071d4d0bf766806a64490176ceaed2d5d046b5f/ebusd-2.1.x/de/vaillant/08.hmu.csv#L27) for the input temperature of the source (in my case I have a brine heatpump).

If you find out additional interesting parameters, please let me know :-)

BR,

mw

**Reply by Bavarialex** [reply]
Great! Got your 8.hmu to get the Integral...  :-) Thanks a lot!

**Comment by JanHBade** [comment]
thanks I will test your files...

**Comment by Bavarialex** [comment]
If you need Flowrate, check hmu WaterThroughput.
It's not in mwildbolz 8.hmu, you have to add manually at the end:
`# Values from UniTower Test Menu,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
*r,,,,,,B514,05,,,IGN:2,,,,,,,,,,,,,,,,,,,,,
r,,WaterThroughput,,,,,2B03FFFF,,,UIN,,l/h,Aktuelle Heizwasser Durchflussrate der Wärmepumpe in Liter pro Stunde,,,,,,,,,,,,,,,,,,`

**Reply by JanHBade** [reply]
Is that the water inside the house? or is that the flow for the coolant of the heatpump (for me goes in a deep hole outside)?

**Reply by Bavarialex** [reply]
It's the throughput of water in litres/hour from outside to inside unit - if you are using a monobloc-
